src/VirtoCommerce.ProfileExperienceApiModule.Data/Middlewares/EmptyMiddleware.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Threading.Tasks;
+using PipelineNet.Middleware;
+using VirtoCommerce.ProfileExperienceApiModule.Data.Aggregates.Vendor;
+
+namespace VirtoCommerce.ProfileExperienceApiModule.Data.Middlewares;
+
+public class EmptyMiddleware: IAsyncMiddleware<VendorAggregate>
```

Comment:
add comments for why this is needed or remove it if not

Reply:
Fixed